tests/notebooks/topic_evaluation.ipynb

```diff
@@ -91,14 +91,489 @@
+       "149  2024-05-28 13:52:30.605484  2024-05-28 13:52:26.182067   \n",
+       "150  2024-04-30 14:43:33.401486  2024-04-30 14:43:29.722961   \n",
+       "151  2024-09-05 08:51:27.530783  2024-09-05 08:51:22.424305   \n",
+       "152  2024-07-08 11:42:26.309643  2024-07-08 11:42:22.455315   \n",
+       "\n",
+       "           userThankedTimestamp approved          llmPromptTimestamp  \\\n",
+       "0    2024-07-29 12:56:42.142827     True  2024-07-29 12:55:16.317499   \n",
+       "1    2024-07-30 14:41:41.500323     True  2024-07-30 14:40:16.941867   \n",
+       "2    2024-07-10 10:50:12.348356     True  2024-07-10 10:48:55.466094   \n",
+       "3    2024-07-04 10:10:46.878537    False  2024-07-04 10:09:51.964797   \n",
+       "4    2024-07-01 10:12:18.961727     True  2024-07-01 10:10:42.207175   \n",
+       "..                          ...      ...                         ...   \n",
+       "148  2024-07-03 09:22:02.639041     True  2024-07-03 09:21:03.425560   \n",
+       "149  2024-05-28 13:52:27.504229     True  2024-05-28 13:50:59.487989   \n",
+       "150  2024-04-30 14:43:33.407613      NaN  2024-04-30 14:43:05.224876   \n",
+       "151  2024-09-05 08:51:24.348777    False  2024-09-05 08:50:24.510631   \n",
+       "152  2024-07-08 11:42:23.967549     True  2024-07-08 11:40:42.720551   \n",
+       "\n",
+       "                   messageId  \\\n",
+       "0    9XYRPT0_GZs.9XYRPT0_GZs   \n",
+       "1    9UY_6vHKveI.9UY_6vHKveI   \n",
+       "2    SAHP2Y9tCRM.SAHP2Y9tCRM   \n",
+       "3    ogTQz8PHcg8.ogTQz8PHcg8   \n",
+       "4    3kPiyGnCrRo.3kPiyGnCrRo   \n",
+       "..                       ...   \n",
+       "148  zbyrDOIyjPs.zbyrDOIyjPs   \n",
+       "149  lEelJyxLwDo.lEelJyxLwDo   \n",
+       "150  1xF2IsPn7fw.1xF2IsPn7fw   \n",
+       "151  2zBPipLX90E.2zBPipLX90E   \n",
+       "152  v8IIvES4L2c.v8IIvES4L2c   \n",
+       "\n",
+       "                                             llmAnswer supervisorMessage  ...  \\\n",
+       "0    <font color=\"#004f88\"><b>Brief Summary:</b> To...              None  ...   \n",
+       "1    <font color=\"#004f88\"><b>Brief Summary:</b> Ad...              None  ...   \n",
+       "2    <font color=\"#004f88\"><b>Brief Summary:</b> <i...              None  ...   \n",
+       "3    <font color=\"#004f88\"><b>Brief Summary:</b> Yo...                    ...   \n",
+       "4    <font color=\"#004f88\"><b>Brief Summary:</b> An...              None  ...   \n",
+       "..                                                 ...               ...  ...   \n",
+       "148  <font color=\"#004f88\"><b>Brief Summary:</b> Th...              None  ...   \n",
+       "149  <font color=\"#004f88\"><b>Brief Summary:</b> <i...              None  ...   \n",
+       "150  Based on the information provided, I will try ...               NaN  ...   \n",
+       "151  <font color=\"#004f88\"><b>Brief Summary:</b> Th...                    ...   \n",
```

Comment:
I think you should clear the outputs of this before merging (also is this not from your eval pr?)